### ISSUE TITLE
Encountered error when cloning repo or downloading the release

### ISSUE THREAD

**Issue by LianSheng197** [danielmiessler/SecLists#667]
RPC failed when trying to cloning repo.
```
git version 2.17.1
```
![Selection_334](https://user-images.githubusercontent.com/18535495/141611323-07b86772-8986-46bb-a4a4-6876aa5703af.png)

Encountered a decompressing error when trying the release version. You can saw that the size of the compressed file is quite small.
```
Ubuntu 18.04.5 LTS
Archive Manager 3.28.0 (An archive manager for GNOME.)
```
![Selection_335](https://user-images.githubusercontent.com/18535495/141611387-abfaa9ea-9461-4d5b-a44f-a77bd2de6da2.png)

Now I try to download the release version from SourceForge, hope this will solve the problem

**Comment by g0tmi1k** [comment]
Odd about `git clone` failing. I wonder if GitHub that day was having issues? Does it still happen?

I can understand why zip fails to open, as that is a incomplete download. Again, possible network issues?

As far as Im aware, there isn't anything hosted on SourceForge?

**Comment by LianSheng197** [comment]
Okay, I tried clone this repo again, this time it can be completed normally. Maybe something rare problem really appeared on GitHub that day :thinking:
In short, on the same day I determined that GitHub Status is all operational, but I did not further check what the problem was. This was my negligence.

As for SourceForge, I downloaded it from here:
https://sourceforge.net/projects/seclists.mirror/

It looks like it is a mirror maintained by SourceForge itself and is not part of SecLists.
